### ISSUE TITLE
Add support for <title> element

### ISSUE THREAD

**Issue by davidruvolo51** [davidruvolo51/rheroicons#9]
When the attribute `aria-hidden` is not present, it is ideal to set a title that describes the icon. This is important if the element should be interactive. However, using the "visually hidden element" may be easier.

- [x] Adjust conversion script for..
    - [x] new argument `title`
    - [x] append svg children with `<title>` when defined by user
    - [x] roxygen documentation for usage
    - [x] roxygen docs for `@param title`
    - [x] test
